pkg/config/environ.go

```diff
@@ -63,4 +63,7 @@ func FromEnv(pArgs *ProgArgs) {
 	if pArgs.RTE.MetricsPort == 0 {
 		pArgs.RTE.MetricsPort = metricssrv.PortFromEnv()
 	}
+	if pArgs.RTE.MetricsIP == "" {
+		pArgs.RTE.MetricsIP = metricssrv.IPFromEnv()
+	}
```

Comment:
the fact we get the metric port from env is a historical accident, but let's be consistent

Reply:
Ack, Thanks!